website/content/docs/commands/monitor.mdx

```diff
@@ -1,47 +1,71 @@
 ---
 layout: docs
-page_title: monitor - Command
+page_title: 'monitor - Vault CLI'
 description: |-
-  The "monitor" command displays the server logs of a Vault server in real
-  time.
+  Display the server logs of a Vault server in real time.
 ---
 
-# monitor
+# `monitor`
 
-The `monitor` command shows a real time display of the server logs of a Vault
+Show a real time display of the server logs of a Vault server
+
+<CodeBlockConfig hideClipboard>
+
+```shell-session
+$ vault monitor [flags]
+
+$ vault monitor [-help | -h]
+```
+
+</CodeBlockConfig>
+
+## Description
+
+`vault monitor` command shows a real time display of the server logs of a Vault
 server. This command accepts a log level as an argument, which can be different
 from the log level that the Vault server was started with.
 
-The `monitor` command honors the `VAULT_ADDR` environment variable. The address
+`vault monitor` honors the `VAULT_ADDR` environment variable. The address
 specified determines the target server that will be monitored.
 
-Note that this command is designed to run indefinitely. It is similar to
-`tail -f` in the Unix world. This command will not exit on its own unless
-it encounters an unexpected error. As a user, you must terminate this
-process yourself to shut it down.
+### Limitations and warnings
 
-If Vault is emitting log messages faster than a receiver can process them, the
-some log lines will be dropped.
+- Note that this command is designed to run indefinitely. It is similar to
+  `tail -f` in the Unix world. This command will not exit on its own unless
+  it encounters an unexpected error. As a user, you must terminate this
+  process yourself to shut it down.
 
-## Examples
+- If Vault is emitting log messages faster than a receiver can process them, the
+  some log lines will be dropped.
 
-Monitor server logs at the `debug` log level:
+## Command arguments
 
-```shell-session
-$ vault monitor -log-level=debug
-```
+- None
 
-## Usage
+## Command options
 
-The following flags are available in addition to the [standard set of
-flags](/vault/docs/commands) included on all commands.
+- None
 
-### Output options
+## Command flags
 
-- `-log-level` `(string: "info")` - Monitor the Vault server at this log level.
-  Valid log levels are (in order of detail) "trace", "debug", "info",
-  "warn", "error". If this option is not specified, "info" is used.
+<br />
 
-- `-log-format` `(string: "standard")` - Format to emit logs.
-  Valid formats are "standard", and "json". 
-  If this option is not specified, "standard" is used.
+@include 'cli/monitor/flags/log-level.mdx'
+
+<br />
+<hr />
+<br />
+
+@include 'cli/monitor/flags/log-format.mdx'
+
+## Standard flags
+
+@include 'cli/standard-settings/all-standard-flags-but-format.mdx'
```

Comment:
This command might actually support the `-format` flag since it's printing info to the screen. I would change this to `all-standard-flags.mdx` for now